src/ui/layout/mct-tree.vue

```diff
@@ -237,20 +244,25 @@ export default {
                 this.searchDeactivated();
             }
         },
-        searchResultItems() {
-            this.setContainerHeight();
-        },
         allTreeItems() {
-            // catches an edge case race condition and when new items are added (ex. folder)
+            // catches an edge case when new items are added (ex. folder)
             if (!this.isLoading) {
                 this.setContainerHeight();
             }
         },
         ancestors() {
             this.observeAncestors();
+        },
+        availableContainerHeight() {
+            this.updatevisibleItems();
```

Comment:
camelCase is not being observed here. Should be `updateVisibleItems`

Reply:
Got it!